Drivers/src/LTC2630.cpp

```diff
@@ -10,9 +10,10 @@
 namespace SolarGators {
 namespace Drivers {
 
-LTC2630::LTC2630(SPI_HandleTypeDef* hspi, OperatingMode mode):hspi_(hspi),mode_(mode),current_val_(0)
+LTC2630::LTC2630(SPI_HandleTypeDef* hspi, GPIO_TypeDef* cs_port, uint16_t cs_pin, OperatingMode mode):
+    hspi_(hspi), cs_port_(cs_port), cs_pin_(cs_pin), mode_(mode), current_val_(0)
 {
-  WriteAndUpdate(0x00);
+//  WriteAndUpdate(0x00);
```

Comment:
Did you mean to delete this?

Reply:
Ahh, forgot about this there needs to be an init function because we can't write anything in the constructor. This is because the clocks are not initialized yet when this is a global variable.

Reply:
Resolved in [3ed4384](https://github.com/John-Carr/SolarGatorsBSP_STM/pull/1/commits/3ed4384ecbe4f4ced0ae289264bb7c14eb0de1c4)